config/blade-icons.php

```diff
@@ -71,6 +71,12 @@
             'class'  => 'fill-current',
         ],
 
+        'wallets' => [
+            'path'   => 'resources/icons/wallets',
```

Comment:
Could just do `<x-ark-icon name="app-wallets.exodus">` I think :thinking:

Reply:
negative

Reply:
actually it does, I forgot to update another reference